spec_finder.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
+import logging
```

Comment:
I would personally not introduce the logging module. It's too heavy weight for CLI output, imo.